palantir-java-format/src/main/java/com/palantir/javaformat/OpsBuilder.java

```diff
@@ -620,6 +631,10 @@ private static int getI(Input.Token token) {
         return newOps.build();
     }
 
+    private static boolean isNonNlsComment(Input.Tok tokAfter) {
+        return tokAfter.isSlashSlashComment() && tokAfter.getText().contains("$NON-NLS");
```

Comment:
May be able to make this more specific, nls comments are required to start with `//$NON-NLS-`, even subsequent numbers, e.g.
```java
String f() {
  return "a" + "b"; //$NON-NLS-1$ //$NON-NLS-2$
}
```

Realistically I'm not sure that makes much difference.